pyvenv.el

```diff
@@ -285,9 +285,19 @@ This is usually the base name of `pyvenv-virtual-env'.")
 (defvar pyvenv-workon-history nil
   "Prompt history for `pyvenv-workon'.")
 
+(defun pyvenv--already-activated (name)
+  "Determine if the virtual environment NAME is already activated."
+  (and (boundp 'pyvenv-virtual-env-name)
```

Comment:
This will always be `t`, because this file is loaded when this function runs.

Reply:
Ah, good point. I copied the logic from my init where the check was being made (potentially) before pyvenv was loaded.